CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Migrate E2E Vulnerability Detector test packages to S3 repository ([#5376](https://github.com/wazuh/wazuh-qa/pull/5376)) \- (Tests)
```

Comment:
```suggestion
- Migrate E2E Vulnerability Detector test packages to S3 repository ([#5376](https://github.com/wazuh/wazuh-qa/pull/5376)) \- (Framework)
```

Reply:
Changed in https://github.com/wazuh/wazuh-qa/pull/5376/commits/6c62908527744914b3c3d228c83fe7911abaac8a